internal/usage/usage.go

```diff
@@ -237,6 +237,7 @@ const (
 	EndpointServiceID                        = "Unique identifier of the private endpoint service for which you want to retrieve a private endpoint."
 	PrivateEndpointIDAzure                   = "Unique identifier of the Azure private endpoint resource."
 	PrivateEndpointIPAddressAzure            = "Private IP address of the private endpoint network interface you created in your Azure VNet."
+	Endpoint                                 = "List of GCP endpoints in the group, of form: endpointName1@ipAddress1,...,endpointNameN@ipAddressN"
```

Comment:
List of GCP endpoints in the group separated by commas, such as:

Reply:
Thanks Julia, I have changed that in the latest commit now.